workshops/networking/1.1-adhoc/README.md

```diff
@@ -0,0 +1,83 @@
+# Exercise 1.1 - Running Ad-hoc commands
```

Comment:
>This doesn't work for me and needs to be corrected. There should be a top level guides/ path that these go under and not mix them in here.

@tima I don't understand the request here, can you be more prescriptive? 

>Also, are there workshop assignments and solution for network automation?

The solution is linked in the Answer Key per lab.  1.1 is an odd duck because the answer is basically provided in the README due to its nature (you are not building a playbook, but rather running commands).